scripts/annotate/check_annotations.py

```diff
@@ -0,0 +1,133 @@
+from omero.gateway import BlitzGateway
+from omero.model import MapAnnotationI
+import pandas
+import sys
+import os
+
+
+def printusage():
+    print('''
+Checks an IDR dataset to make sure each image/well has an annotation.
+Prints the images/wells which don't have any annotations to stdout.
+
+If an annotation.csv is provided the images will be checked if they have
+a corresponding entry in the annotation.csv. If it is omitted the images
+will simply be check if they have at least one annotation attached.
```

Comment:
```suggestion
will simply be checked if they have at least one annotation attached.
```

Maybe it should say ``the images will simply pass the test if...`` ?

Reply:
I understand now that one is supposed to run the option **with annotations.csv** before one makes any annotation steps, to ease the workflow. Please add this to the help text.